libs/engine/includes/Serialization/Serializer.hpp

```diff
@@ -39,7 +39,7 @@ class Serializer {
     {
         if (str.size() > std::numeric_limits<SizeType>::max())
             throw std::runtime_error("String too long");
-        SizeType size = str.size();
+        auto size = static_cast<SizeType>(str.size());
```

Comment:
```suggestion
        SizeType size(str.size());
```
C’est beaucoup trop long les static cast…

Reply:
C:\development\r-type\libs\engine\.\includes\Serialization/Serializer.hpp(42): warning C4267: 'initialisation' : conversion de 'size_t' en 'uint8_t', perte possible de données
c'est pour ça que j'ai fais un static cast sinon ça me fait ça